### PR TITLE
Pandas accumulator

### PR DESCRIPTION
**Pull request by lucalavezzo**
Finished @chrispap95's pandas accumulator and implemented it in our coffea code.
I've tested the script after implementing it, and all histograms are exactly identical as they are before.
This removes the need for the merger.py script, as we can rely on coffea to merge the chunks' accumulators, each of which is a pandas dataframe.
The metadata and event variables are saved through different accumulators, but are then saved together in an hdf5 file, as before, in the condor_SUEP_WS.py script.

### CODE REVIEW

File: condor_SUEP_WS.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 # SUEP Repo Specific
 from workflows import SUEP_coffea, merger
+from workflows import pandas_utils
```
Comment (chadfreer) [inline condor_SUEP_WS.py]: Move to line above

**Review by chadfreer — COMMENTED** [review]
Move this to a working branch. Looks good

File: condor_SUEP_WS.py (hunk 1 above)
Comment (lucalavezzo) [reply condor_SUEP_WS.py]: done

**Review by chrispap95 — APPROVED** [review]
Looks good to me

**Pull request merged**